.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: java
+before_script:
+  - unset _JAVA_OPTIONS
+  - cd finish
+script:
+  - mvn clean install
+after_script:
+  - build=$(grep "Open Liberty" target/liberty/wlp/usr/servers/defaultServer/logs/console.log | cut -d' ' -f5 | cut -d')' -f1 ); release=$( echo "$build" | cut -d'/' -f1); number=$( echo "$build" | cut -d'/' -f2); echo -e "\n"; mvnOutput=$(mvn install) 2>&1; if ! echo "$mvnOutput" | grep -iq "compilation"; then echo -e  "\033[1;34mOpen Liberty release:\033[0m \033[1;36m$release\033[0m"; echo -e "\033[1;34mOpen Liberty build number:\033[0m \033[1;36m$number\033[0m"; else echo -e "\033[0;31mThere is a compilation error.\033[0m"; fi; echo -e "\033[1;34mJava version:\033[0m"; jv=$(java -version 2>&1); while read -r line; do echo -e "\033[1;36m$line\033[0m"; done <<< "$jv"; 
```

Comment:
We'd get the java version info from the server logs instead of from the system, as they may be different: 

echo -e "\033[1;34mJava version:\033[0m"; 
jv=$(java -version 2>&1); 
while read -r line; 
do echo -e "\033[1;36m$line\033[0m"; 
done <<< "$jv";

Reply:
fixed

Reply:
Ok